zng/docs/spec.md

```diff
@@ -219,11 +250,11 @@ A record typedef creates a new type ID equal to the next stream type ID
 with the following structure:
 ```
 ----------------------------------------------------------
-|0x80|<nfields>|<field1><type-id-1><field2><type-id-2>...|
+|0xf0|<nfields>|<field1><type-id-1><field2><type-id-2>...|
```

Comment:
should `nfields` be `ncolumns`? and `field1` be `fieldname` ?

Reply:
Ok, I did `ncolumns` and `name1`...